code/modules/reagents/reagent_containers/cups/tube.dm

```diff
@@ -0,0 +1,16 @@
+/obj/item/reagent_containers/cup/tube
```

Comment:
Why is this a subtype of cup?

Reply:
I didn't check, but beakers and bottles are also cups, I don't see any reason why these should be different. You can drink from test tube, it's an open container. Just default cup settings fit this.

Reply:
If beakers and bottles are cups fine but they also should not be

Reply:
I think cup is just a generic open container, rather than a cup for tea. Buckets are cups too.

Reply:
They should not be

Reply:
cursed